packages/meditrak-server/src/tests/dhis/EventBuilder.test.js

```diff
@@ -126,49 +126,4 @@ describe('EventBuilder', () => {
       });
     });
   });
-
-  describe('Timezone awareness', () => {
```

Comment:
I've removed these tests because they're testing that the timezone aware value converts correctly, which we are in the process of removing